pkgs/development/compilers/cudatoolkit/stdenv.nix

```diff
@@ -0,0 +1,10 @@
+{ overrideCC
+, wrapCCWith
+, stdenv
+, compatibleStdenv
+}:
+
+overrideCC stdenv (wrapCCWith rec {
+  cc = compatibleStdenv.cc.cc;
+  libcxx = stdenv.cc.cc.lib;
+})
```

Comment:
I've never used `wrapCCWith` and `overrideCC` before, so I've no idea if I'm doing this correctly. I'm also probably abusing the `libcxx` argument, I guess it's a hatch for clang, but it did the job for me somehow

I'd really appreciate a comment from @FRidh who had suggested `overrideCC` at some point